src/System.Security.Cryptography.Cng/src/System/Security/Cryptography/RSACng.ImportExport.cs

```diff
@@ -94,7 +97,15 @@ public override void ImportParameters(RSAParameters parameters)
                     Debug.Assert(offset == blobSize, "offset == blobSize");
                 }
                 CngKeyBlobFormat blobFormat = includePrivate ? s_rsaPrivateBlob : s_rsaPublicBlob;
-                Key = CngKey.Import(rsaBlob, blobFormat);
+
+                CngKey newKey = CngKey.Import(rsaBlob, blobFormat);
+                CngExportPolicies exportPolicy = newKey.ExportPolicy | CngExportPolicies.AllowPlaintextExport;
+                using (SafeNCryptKeyHandle newKeyHandle = newKey.Handle)
```

Comment:
Yeah, the Handle property duplicates the SafeHandle so it has to be disposed anyway, so it was better to do it deterministically.

Having said that, I think I’m going to get rid of the need to Duplicate entirely and tighten the code by exposing an internal ExportPolicy setter in CngKey. I’ve just been waiting to see if there’s other feedback to incorporate.